config/features/flags.go

```diff
@@ -54,6 +54,10 @@ var (
 		Usage: `WARNING: gRPC API will still be supported for some time, most likely until v8 in 2026, but will be eventually removed in favor of REST API.
 		Disables displaying logs for newly connected grpc clients.`,
 	}
+	safeHeadFCU = &cli.BoolFlag{
+		Name:  "safe-head-fcu",
+		Usage: "Enables safe head block instead of justified block in FCU updates.",
```

Comment:
Can you spell out FCU in the description? Users may not know what FCU is. Is there more we can add? Not sure what the implications are from this minimal description.

Reply:
Is this flag meant to be temporary?

Reply:
It should not be a feature flag, rather than a common flag